app/docker/shared-build-env.dockerfile

```diff
@@ -52,7 +52,7 @@ RUN ./src/webmodels/gradlew
 
 # Install Node dependencies
 COPY package.json .
-RUN npm install
```

Comment:
This is unrelated, but it makes the TeamCity build logs *massively* less verbose. Like, thousands of lines less.